### PR TITLE
Implement 'team remove-user' command

### PR DESCRIPTION
**Pull request by aguerra**
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa/321)

### CODE REVIEW

File: pkg/client/cmd/team.go [hunk 1]
```diff
@@ -45,12 +45,25 @@ You need to create a user before use this command.`,
...
 }
 
+var teamRemoveUserCmd = &cobra.Command{
+	Use:   "remove-user",
+	Short: "Remove a member of a team",
+	Long: `Remove a member of a team.
+
+You can remove an user of a team with:
+
+  $ teresa team remove-user --user john.doe@foodomain.com --team foo
```
Comment (yagonobre) [inline pkg/client/cmd/team.go]: Maybe user can be an arg instead of a flag

Comment (yagonobre) [inline pkg/client/cmd/team.go]: Maybe user can be an arg instead of a flag

File: pkg/protobuf/team/team.proto [hunk 2]
```diff
@@ -19,6 +20,11 @@ message AddUserRequest {
     string user = 2;
 }
 
+message RemoveUserRequest {
+    string name = 1;
```
Comment (yagonobre) [inline pkg/protobuf/team/team.proto]: Maybe `team_name`

File: pkg/client/cmd/team.go (hunk 1 above)
Comment (aguerra) [reply pkg/client/cmd/team.go]: I think that for now we need consistency with `add-user` but it is an interesting option for another PR.

Comment (yagonobre) [reply pkg/client/cmd/team.go]: Nice :)

**Pull request merged**